resources/metadata/transportRequestUploadSOLMAN.yaml

```diff
@@ -74,6 +75,7 @@ spec:
           - name: commonPipelineEnvironment
             param: custom/transportRequestId
         type: string
+        mandatory: true
```

Comment:
I'm not sure if that is appropriate here. To my understanding this is used for a check by the corba framework and it might be that we fail if the parameter is not handed over as command line argument. Not fully sure, but should be checked.

Reply:
It is appropriate as long as the git repo isnt scanned here.